components/core/Cargo.toml

```diff
@@ -37,7 +37,8 @@ typemap = "*"
 url = "*"
 
 [target.'cfg(not(windows))'.dependencies]
-users = "*"
+# Pulling from git until https://github.com/nix-rust/nix/commit/3c2107bdc221a90b02d3f7118dd96f3496762cd0 is released
+nix = { git = "https://github.com/nix-rust/nix.git" }
```

Comment:
Might be worth specifying the exact commit here to be explicit:

```toml
nix = {  git = "https://github.com/nix-rust/nix.git", rev = "3c2107bdc221a90b02d3f7118dd96f3496762cd0" }
```

Reply:
Addressed in https://github.com/habitat-sh/habitat/pull/7620/commits/ab95f2a3384489c9333127f010bcb2d0688dcc09